### DISCUSSION TITLE
Reembolso e cancelamento de licença

### DISCUSSION THREAD

**Discussion by camiladsantosvidal** [aclap-dev/video-downloadhelper#1358] [License & Purchase requests]
### What's your issue?

Olá! gostaria de cancelar e pedir um reembolso da minha compra de licença, pois não consegui baixar os vídeos que eu preciso

### The 10 first digits (only 10) of your license number or email address

a1ec6eafd5

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Reply by camiladsantosvidal** [reply]
muito obrigada!

**Comment by artedocuidar2019-svg** [comment]
Olá, boa tarde! gostaria de cancelar e pedir um reembolso da minha compra de licença, pois não consegui baixar os vídeos que eu preciso

**Reply by jcfrog** [reply]
Hi,
please send us an email at licenses@downloadhelper.net

**Comment by beatriz089d-sudo** [comment]
I want to cancel my purchase. I didn't see the correct amount; I thought it was in reais, but it's in dollars, and I can't afford to pay that. I bought it today with my card and haven't used it yet.
What do I do?

**Reply by jcfrog** [reply]
I sent you an email
